jenkins-agent/maven/Dockerfile

```diff
@@ -2,11 +2,13 @@ ARG RUNTIME
 ARG REGISTRY_REPO
```

Comment:
maybe set default value? ref: https://docs.docker.com/reference/build-checks/invalid-default-arg-in-from/